src/Orleans/Utils/IKeyedServiceCollection.cs

```diff
@@ -0,0 +1,37 @@
+﻿
+using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Orleans
```

Comment:
Most things in utils are either Orleans or Orleans.runtime.
Most Orleans public apis,are in these namespaces.